lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -11,16 +12,21 @@ import type {
   VersionsProjectResult,
 } from './types';
 
+function isFullUrl(registryUrl: string): boolean {
+  const regex = regEx(/(http|https):[\\/]{2,}.*[\\/][a-z]+/);
```

Comment:
What exactly is this testing? That there's some path?

Reply:
yeah, the idea is to check whether it's sth like https://my.server.com/ or baseurl + path like https://autohub.example.com/api/galaxy/whatever. if there's a better regex, i'll gladly change it.

Reply:
So if it's a plain hostname like `my.server.com` the you assume you need to add `/api/v2/collections` to it but if it has any path at all (e.g. `my.server.com/foo`) then you assume it does not need anything added to it?

Reply:
exactly.